src/app/Modal/DeleteWarningModal.tsx

```diff
@@ -46,62 +46,58 @@ export interface DeleteWarningProps {
   visible: boolean;
   onAccept: () => void;
   onClose: () => void;
-  setShowDialog: () => void;
-  checkbox?: boolean;
-  setCheckbox?: React.Dispatch<React.SetStateAction<boolean>>;
+  disableDialog: () => void;
 }
 
-export const DeleteWarningModal = ({ warningType , items, visible, onAccept, onClose, setShowDialog, checkbox, setCheckbox}: DeleteWarningProps): JSX.Element => {
+export const DeleteWarningModal = ({ warningType , items, visible, onAccept, onClose, disableDialog}: DeleteWarningProps): JSX.Element => {
   const [doNotAsk, setDoNotAsk] = useState(false);
   const realWarningType = getFromWarningMap(warningType);
 
-  const description = `${realWarningType?.description}${(typeof items === 'undefined' || items.length <= 1) ?  "":"s"}: [${items?.join(", ")}]. ${realWarningType?.descriptionPartTwo}`
+  const description = 
+    <TextContent>
+      <Text>
+        {realWarningType?.description}{(typeof items === 'undefined' || items.length <= 1) ?  "":"s"}: [{items?.join(", ")}]
```

Comment:
The `typeof` check here is a little unconventional. Does `items?.length <= 1` fit the need here?